### PR TITLE
Added the changes to showcase the features on landing page.

### PR DESCRIPTION
**Pull request by rishup132**
Added the changes to showcase the features on landing page.

### CODE REVIEW

File: src/components/LandingPage/components/Features/Features.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import { useEffect, useRef, useState } from 'react';
+
+interface FeatureData {
+  icon: React.ReactElement,
+  title: string,
+  description: string,
+  imageLight: string
+}
+
+const items: Array<FeatureData> = [
```
Comment (pankaj3108) [inline src/components/LandingPage/components/Features/Features.tsx:17]: Move This static variable to different folder.

File: src/components/LandingPage/components/Features/Features.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    icon: <FolderCopyIcon />,
+    title: 'Multi-Document Integration & Information Retrieval',
+    description:
+      'Solace allows users to work with multiple documents, cross-reference information, and easily search for content. \
+      Contextual explanations, accurate answers, and comparison tools enhance knowledge retrieval from the full set of documents.',
+    imageLight: '/src/assets/features/Illustration - Experiment lab.svg'
+  },
+];
+
+export default function Features() {
```
Comment (pankaj3108) [inline src/components/LandingPage/components/Features/Features.tsx:54]: Make declaration consistent for all Components.

File: src/components/LandingPage/components/Features/Features.tsx (hunk 1 above)
Comment (rishup132) [reply src/components/LandingPage/components/Features/Features.tsx]: Done.

File: src/components/LandingPage/components/Features/Features.tsx (hunk 2 above)
Comment (rishup132) [reply src/components/LandingPage/components/Features/Features.tsx]: Done.

**Pull request merged**